services/github/github-tag.service.js

```diff
@@ -15,9 +15,10 @@ const {
 module.exports = class GithubTag extends LegacyService {
   static registerLegacyRouteHandler({ camp, cache, githubApiProvider }) {
     camp.route(
-      /^\/github\/tag(-?pre)?\/([^/]+)\/([^/]+)\.(svg|png|gif|jpg|json)$/,
+      /^\/github\/(tag-?pre|tag-date|tag)\/([^/]+)\/([^/]+)\.(svg|png|gif|jpg|json)$/,
```

Comment:
Do you think it would be better to make the `-` required for `tag-pre`?
```diff
-/^\/github\/(tag-?pre|tag-date|tag)\/([^/]+)\/([^/]+)\.(svg|png|gif|jpg|json)$/
+/^\/github\/(tag-pre|tag-date|tag)\/([^/]+)\/([^/]+)\.(svg|png|gif|jpg|json)$/
```
or an alternative:
```regexp
/^\/github\/tag(?:-(pre|date))?\/([^/]+)\/([^/]+)\.(svg|png|gif|jpg|json)$/ (- required)
/^\/github\/tag(?:-?(pre|date))?\/([^/]+)\/([^/]+)\.(svg|png|gif|jpg|json)$/ (- optional)
```

Reply:
I was keeping this to avoid making a breaking change:
https://github.com/badges/shields/blob/7ea540d3c67b2f98a97fd9651296c6fb717d528b/services/github/github-tag.service.js#L18

..but now you mention it, one of the issues here was we've never put an example of this on the home page so I doubt there is much usage in the wild. It was only added quite recently. Lets change it now :+1: